concepts/Code_status/carePlan_getItemValues.Rmd

```diff
@@ -0,0 +1,34 @@
+---
+title: "carePlan_getItemValues"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+Connect to database.
+
+```{r}
+require("RPostgreSQL")
+drv <- dbDriver("PostgreSQL")
+con <- dbConnect(drv, dbname = "eicu",
+                 host = "localhost", port = 5432,
+                 user = "josephpark", password = rstudioapi::askForPassword("Database password"),
+                 options = "--search_path=eicu")
+dbExistsTable(con, "patient")
+```
+
+Write cplitemvalues to file "possible_status" for inspection on which items indicate patient code status.
+All cplitemvalues that were chosen were in cplgroup = 'Care Limitation' (all cplitemvalues in this cplgroup was used),
+  except cplitemvalue = 'End of life', which was in cplgroup = 'Ordered Protocols'.
+
+```{r, connection = con}
+library(dplyr)
```

Comment:
The SQL is only 4 lines long, so it seems overkill to include a whole RMarkdown file. Couldn't we just use the SQL, which is more consistent with the usual approach?

Reply:
We could do either.  @psy01212 did it in R, I asked him to use the translate facility in `dplyr` to have that option available.

Reply:
I have included both versions, although I have done the analysis through R, for consistency since most of the MIMIC code repo is in SQL, so having an .sql file might be useful as a template.